### ISSUE TITLE
AutocompleteInput suggestion list location

### ISSUE THREAD

**Issue by selenyillar** [marmelab/react-admin#2961]
AutocompleteInput suggestion list location still have these bugs after installing the latest release. (2.7.3) 

If the suggestion list opens upward, when you search something, the list sticks to the top as it narrows.
Also the list panel stays fixed as you scroll down/up the page.

<img width="990" alt="screen shot 2019-02-06 at 10 37 58" src="https://user-images.githubusercontent.com/44598915/52315813-e9f21f80-29fb-11e9-8b26-941d24015468.png">
 

**Environment**

- React-admin version: 2.7.3
- React version: 16.8.3
- Browser: Chrome Version 72.0.3626.121

**Comment by Kmaschta** [comment]
What is your browser version please?

**Comment by selenyillar** [comment]
Updated my comment.

**Comment by marquesgabriel** [comment]
Hello, I'm facing this same situation. The AutocompleteInput's suggestions inside an ArrayInput floats away from the input. I'll insert a gif as an example:

--------------------------

![diagnose](https://user-images.githubusercontent.com/15978414/54783545-69306100-4c00-11e9-8177-dd7091c7ccfb.gif)

And here is the piece of code

```
<ArrayInput label="Localizações e Diagnósticos" source="Pathologies">
      <SimpleFormIterator>
        <ReferenceInput label="Localização da Lesão" source="WoundLocaleId" reference="lesoes" filterToQuery={val => ({ localization: val })}>
          <AutocompleteInput inputValueMatcher={() => null} optionText="localization"/>
        </ReferenceInput>
        <ReferenceInput label="Diagnóstico" source="diagnoseId" reference="diagnosticos" filterToQuery={val => ({ description: val })}>
          <AutocompleteInput inputValueMatcher={() => null} optionText="description"/>
        </ReferenceInput>
      </SimpleFormIterator>
</ArrayInput>
```

--------------------------

## Environment
 - React-admin version: 2.8.2
 - React version: 16.8.4
 - Browser:
    - Chrome Version 73.0.3683.86
    - Firefox Version 68.0a1

PS: I ensured the packages were fully updated. I cleaned the cache and removed the node_modules folder and then installed all of them

**Comment by FACOLOMBANI** [comment]
The problem comes from de Popper component inside of the AutocompleteInput. Try adding to the AutocompleteInput the options below:

`<AutocompleteInput
                    inputValueMatcher={() => null}
                    optionText="localization"
                    options={{
                    suggestionsContainerProps: {
                        disablePortal: true,
                        modifiers: {
                            keepTogether: {
                                enabled: true
                            }
                        }
                    }
                }}/>`

**Comment by marquesgabriel** [comment]
The disablePortal option positioned it wrong
![autocompeltebuggy](https://user-images.githubusercontent.com/15978414/55072663-70c19100-506a-11e9-9e3f-2c568050015b.gif)
 :cry:

**Comment by FACOLOMBANI** [comment]
Did you add the modifiers as well?

**Comment by marquesgabriel** [comment]
Yes. Tried just like you sent, without the disablePortal option and some others also. keepTogether must be true to work properly

**Comment by FACOLOMBANI** [comment]
Can you try those config in an AutocompleteInput outside the ArrayInput and SimpleformIterator iterator to see if its a compatibility problem with the ArrayInput and SimpleformIterator?

**Comment by marquesgabriel** [comment]
I tried the AutoCompleteInput on a filter component also. Same result. I think the problem is related to this issue that is already fixed - https://github.com/marmelab/react-admin/issues/2259
I cloned React-Admin repo and I'll make some experiments to see if i can find the origin of this behaviour

**Comment by FACOLOMBANI** [comment]
I did the same because i had the same issue, look a this issue, the second comment https://github.com/marmelab/react-admin/issues/2370

The problem comes from the popper rendering outside its bouderies. In my case the wysiwyg i had rendered after the autocomplete input, so the popper element was taking the initial position of the autocomplete input to render. So i tested several modifiers and the solution i sent to you was the correct for my case. 

In your case, it has to be another condition, but it still comes from the popper rendering outside the bouderies.

**Comment by marquesgabriel** [comment]
Thanks for all the help, i'll try some other things and if i succeed fixing the problem i post it here

**Comment by komik966** [comment]
![Peek 2019-04-04 10-36](https://user-images.githubusercontent.com/8155563/55541734-dabee380-56c5-11e9-87c4-0959142a7931.gif)

**Comment by Kmaschta** [comment]
Related to #3031

**Comment by fzaninotto** [comment]
Fixed by #3683
